@here/harp-datasource-protocol/lib/Expr.ts

```diff
@@ -195,31 +197,34 @@ interface ReferenceResolverState {
 }
 
 /**
- * The evaluation scope of an [[Expr]].
+ * The evaluation scope of an {@link Expr}.
  */
 export enum ExprScope {
     /**
-     * The scope of an [[Expr]] used as value of an attribute.
+     * The scope of an {@link Expr} used as value of an attribute.
      */
     Value,
 
     /**
-     * The scope of an [[Expr]] used in a [[Technique]] `when` condition.
+     * The scope of an {@link Expr} used in a [[Technique]] `when` condition.
      */
     Condition,
 
     /**
-     * The scope of an [[Expr]] used as dynamic property attribute value.
+     * The scope of an {@link Expr} used as dynamic property attribute value.
      */
     Dynamic
 }
 
+// tslint:disable:max-line-length
 /**
- * Abstract class defining a shape of a [[Theme]]'s expression
+ * Abstract class representing the
+ * {@link https://github.com/heremaps/harp.gl/blob/master/%40here/harp-datasource-protocol/StyleExpressions.md | style expressions}
```

Comment:
can this link to a local file? You are anyway in the `harp-datasource-protocol` lib